src/components/Transaction.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { memo } from "react";
+
```

Comment:
try to keep the number of whitespaces to the minimum.

i.e. not need for newlines between imports and between brackets `import React, {memo} from 'react'`